test/unit_tests/modules/util/strings-spec.js

```diff
@@ -40,6 +40,16 @@ describe( 'Strings stringMatch()', function() {
       .to.be.true;
   } );
 
+  it( 'should return true when testing matching strings with differing casing',
```

Comment:
Thanks! I wasn't sure what the behavior was going to be, so this doubles as documentation. :+1: